src/js/stores/UnitHealthStore.js

```diff
@@ -94,7 +93,7 @@ const UnitHealthStore = Store.createStore({
   },
 
   shouldPoll: function () {
-    return !_.isEmpty(this.listeners(HEALTH_UNITS_CHANGE));
+    return (this.listeners(HEALTH_UNITS_CHANGE).length === 0);
```

Comment:
same here. the original code checks if it's *not* empty, this checks if it *is* empty.